
class ConnectionNotFoundException(BaseException):
    def __init__(self):
        pass